gate-core/src/main/groovy/com/netflix/spinnaker/gate/filters/FiatSessionFilter.groovy

```diff
@@ -75,7 +75,11 @@ class FiatSessionFilter implements Filter {
             session.invalidate()
             SecurityContextHolder.clearContext()
           }
+        } else {
+          log.debug("Unable to get permission from from permission evaluator for user: ${user}.")
```

Comment:
nit: `key=value` rather than `key: value` -- So here that would be `user=${user}`.

Reply:
fixed.